src/plugins/data/public/autocomplete/providers/value_suggestion_provider.ts

```diff
@@ -85,6 +97,15 @@ export const setupValueSuggestionProvider = (
       : undefined;
     const filterQuery = timeFilter ? buildQueryFromFilters([timeFilter], indexPattern).filter : [];
     const filters = [...(boolFilter ? boolFilter : []), ...filterQuery];
-    return await requestSuggestions(title, field, query, filters, signal);
+    try {
+      return await requestSuggestions(title, field, query, filters, signal);
+    } catch (e) {
+      if (!signal?.aborted) {
+        usageCollector?.trackError();
+      }
+      // Remove rejected results from memoize cache
+      requestSuggestions.cache.delete(resolver(title, field, query, filters));
```

Comment:
Wouldn't calling resolver here also call `usageCollector?.trackCall();` one more time? 
I think this isn't what we want?

Reply:
Oops!!!

Thanks for the catch!